habitat_sim/simulator.py

```diff
@@ -104,7 +104,16 @@ def _config_pathfinder(self, config: Configuration):
         if "navmesh" in config.sim_cfg.scene.filepaths:
             navmesh_filenname = config.sim_cfg.scene.filepaths["navmesh"]
         else:
-            navmesh_filenname = osp.splitext(config.sim_cfg.scene.id)[0] + ".navmesh"
+            scene_basename = osp.basename(config.sim_cfg.scene.id)
```

Comment:
Add a short explanatory comment, maybe? Took me a while to understand where it looks and why, and had to crosscheck with the Replica filesystem layout.